examples/osc/expected-output.1

```diff
@@ -90,9 +88,9 @@ help opt\-baz1
 .TP
 \fB\-\-opt\-baz2\fR \fI\,OPT_BAZ2\/\fR
 help opt\-baz2
-
 .RE
 
+
```

Comment:
This additional space is something I'm not sure how to remove @dmach, it shouldn't matter much but feel free to review (or fix once merged).